ouroboros-network-protocols/ouroboros-network-protocols.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:       3.0
 
 name:                   ouroboros-network-protocols
-version:                0.3.0.0
+version:                0.4.0.0
```

Comment:
Please add an entry in `CHANGELOG.md` file.  Even if it's just an empty list, it's useful to record that, so we won't look into that in the future.